lib/collection/src/operations/types.rs

```diff
@@ -88,6 +88,8 @@ pub struct CollectionInfo {
 #[derive(Debug, Deserialize, Serialize, JsonSchema, PartialEq)]
 #[serde(rename_all = "snake_case")]
 pub enum UpdateStatus {
+    /// Unknown update status
+    Unknown,
```

Comment:
This is going to change the API. Maybe we better add another match to the conversion instad?